cli/src/main/kotlin/OrtServerMain.kt

```diff
@@ -58,6 +59,10 @@ class OrtServerMain : SuspendingNoOpCliktCommand(COMMAND_NAME) {
             message = { "$commandName CLI version $it" }
         )
     }
+
+    override fun help(context: Context) = """
+        The ORT Server Client (osc) CLI is a command line interface to interact with an ORT Server instance.
```

Comment:
To avoid "CLI is a command line interface", maybe just drop "CLI"? Or say "The ORT Server Client (OSC) is a Command Line Interface (CLI) to interact with an ORT Server instance."